docs/faq.md

```diff
@@ -344,6 +345,12 @@ See more at [Using Google Container Registry (GCR) with Minikube](https://ryanes
 
 Jib currently requires configuring your build tool to use the appropriate [Java networking properties](https://docs.oracle.com/javase/8/docs/api/java/net/doc-files/net-properties.html) (`https.proxyHost`, `https.proxyPort`, `https.proxyUser`, `https.proxyPassword`).
 
+### What should I do when the registry responds with Forbidden or DENIED?
+
+If the registry returns "403 Forbidden" or `"code":"DENIED"`, it often means Jib successfully authenticated using credentials you provided but the credentials do not have permissions to pull or push images. Make sure your account/role has the permissions to do the operation.
```

Comment:
"403 Forbidden" vs `"403 Forbidden"` ? since we're doing it for the next `"code":"DENIED"`